service/src/main/java/bio/terra/cbas/dao/MethodVersionDao.java

```diff
@@ -23,8 +23,8 @@ public MethodVersionDao(NamedParameterJdbcTemplate jdbcTemplate) {
 
   public int createMethodVersion(MethodVersion methodVersion) {
     return jdbcTemplate.update(
-        "insert into method_version (method_version_id, method_id, method_version_name, method_version_description, method_version_created, method_version_last_run_set_id, method_version_url) "
-            + "values (:methodVersionId, :methodId, :name, :description, :created, :lastRunSetId, :url)",
+        "insert into method_version (method_version_id, method_id, method_version_name, method_version_description, method_version_created, method_version_last_run_set_id, method_version_url, method_version_original_workspace_id) "
```

Comment:
small nit: Here and in other tables, should this be simply called `workspace_id` or `<table_name>_workspace_id`? Is there a reason to add `original` in the name?

Reply:
Won't the workspace IDs in this column always belong to the "original" workspace (irrespective of it was from source workspace or current workspace)? I guess maybe I am not understanding the nuance here 🤔

Reply:
So we won't be storing the workspace id for the current workspace when new methods are imported or when new run sets are launched (in the cloned workspace)?

Reply:
We should get both - when we import a new method or start a new run set, this value gets set to the current workspace. Then when we clone it, it stays set at the original workspace rather than being updated to the new one.

Reply:
I see. Thank you for the clarification! 👍